src/app/Http/Controllers/AdminChoiceAddController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\BigQuestion;
+use App\Question;
+use App\Choice;
+
+class AdminChoiceAddController extends Controller
+{
+    public function index(Request $request, $big_question_id, $question_id)
```

Comment:
Request利用していなければ削除しても良いかと思います。
```suggestion
    public function index($big_question_id, $question_id)
```